src/components/Graphs/DoughnutGraph/DoughnutGraph.tsx

```diff
@@ -0,0 +1,88 @@
+import { GraphCard, HeaderLine, HeaderWrapper, StyledHeader } from "../style";
+import {
+  Dot,
+  GraphContainer,
+  LabelContainer,
+  LabelsContainer,
+  StyledPrecentage,
+  StyledValueText,
+  Wrapper,
+} from "./style";
+import { PieChart, Pie, Cell, Legend } from "recharts";
+import {
+  Props as LegendProps,
+  ContentType as LegendType,
+} from "recharts/types/component/DefaultLegendContent";
+type DataProps = {
+  name: string;
+  value: number;
+};
+interface DoughnutGraphProps {
+  header?: string;
+  data?: DataProps[];
+  colorPalette: string[];
+  innerText: string;
+}
+
+export const DoughnutGraph: React.FC<DoughnutGraphProps> = ({
+  header,
+  data,
+  innerText,
+  colorPalette,
+}) => {
+  const renderLegend: LegendType = (props: LegendProps) => {
```

Comment:
Declare this function somewhere else and render it here. The best way would be to declare this compnent under src/components/Graphs/DoughnutGraph/components/RenderLegend/RenderLegend.tsx